superset/migrations/versions/2025-03-07_08-35_d4d5115bdeb0_add_extensions_table.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add extensions table
+
+Revision ID: d4d5115bdeb0
+Revises: 74ad1125881c
+Create Date: 2025-03-07 08:35:14.112691
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "d4d5115bdeb0"
+down_revision = "74ad1125881c"
+
+from uuid import uuid4  # noqa: E402
+
+import sqlalchemy as sa  # noqa: E402
+from alembic import op  # noqa: E402
+from sqlalchemy_utils import UUIDType  # noqa: E402
+
+
+def upgrade():
```

Comment:
@villebro Could you use the [migration utility functions](https://github.com/apache/superset/blob/master/superset/migrations/shared/utils.py#L218)?